src/modules/user/service.js

```diff
@@ -29,6 +29,16 @@ const createUser = async (data) => {
   }
 };
 
+const getUser = async (email) => {
+  const user = await db.User.findOne({ where: { email } });
+  if (user) {
```

Comment:
Actually you doing extra work
You can just return `db.User.findOne` even without `await`
And then do await for getUser func it will return user object or undefined